ApiBundle/Controller/ContentController.php

```diff
@@ -101,10 +101,32 @@ public function showOrCreateAction(Request $request, $contentId)
     public function listAction(Request $request)
     {
         $contentType = $request->get('content_type');
+        $columns = $request->get('columns');
```

Comment:
The contentController, ContentController and SiteController use the different method in repository which add additional filters so i can't use the trait HandleRequestDataTable